views/emails/studentMail-2.ejs

```diff
@@ -29,6 +29,9 @@
   </p>
   <p>
     <%= signature %> Santé Psy Étudiant
+  </p>  
+  <p class="footer">
+    si tu ne souhaites plus recevoir d'email de notre part: <a href="<%= unregister %>">Se désinscrire</a>
```

Comment:
We may add caps and space before ":"
```suggestion
    Si tu ne souhaites plus recevoir d'email de notre part : <a href="<%= unregister %>">Se désinscrire</a>
```
If suggestion is accepted: to be done for all emails.